storage/fanout/fanout_test.go

```diff
@@ -79,7 +79,7 @@ func TestSelectSorted(t *testing.T) {
 	matcher, err := labels.NewMatcher(labels.MatchEqual, model.MetricNameLabel, "a")
 	testutil.Ok(t, err)
 
-	seriesSet, _, err := querier.SelectSorted(nil, matcher)
+	seriesSet, _, err := querier.Select(false, nil, matcher)
```

Comment:
Shouldn't this be true so the test is deterministic?

Reply:
Ah, nice ! missed this